### PR TITLE
Update migrate-from-e3.md

### PR DESCRIPTION
**Pull request by msbemba**
added migration information about office installs

### CODE REVIEW

**Comment by msbemba** [comment]
Issue# https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1781

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/business/migrate-from-e3.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ You don't need to do any changes to your current subscription or data before mig
 - Subscription configuration, such as DNS records and domain names.
 - User and group accounts and authentication settings, such as multi factor authentication or conditional access policies.
 - Productivity service configurations and their data, such as Teams, Exchange Online mailboxes, SharePoint Online sites, OneDrive for Business folders, and OneNote notebooks.
+- Office Applications will scale automatically.  Office 365 modern licensing will check the user’s license assignment every 72 hours and will convert Office apps to the version that matches the user subscription.
```
Comment (JohanFreelancer9) [inline microsoft-365/business/migrate-from-e3.md]:
```suggestion
- Office applications will scale automatically. Office 365 modern licensing will check the user’s license assignment every 72 hours and will convert Office applications to the version that matches the user subscription.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. This is based on issue #1781.

Thank you.

cc: @kenwith

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**